src/server/origin/cph-number/controller.js

```diff
@@ -48,13 +51,17 @@ export const postController = {
       })
     }
 
-    req.yar.set('cphNumber', input)
+    req.yar.set('origin', {
+      ...req.yar.get('origin'),
+      cphNumber: input
+    })
 
-    return res.redirect('/origin-address')
+    return res.redirect(nextPage ?? '/origin/address')
```

Comment:
I think this is broken, because in all cases the nextPage hidden field is going to be "" if a redirect_uri is not provided.

so the null coalescing causes this to return a 302 with location of `''`